cargo-nextest/src/dispatch.rs

```diff
@@ -108,6 +111,13 @@ enum Command {
             value_name = "FMT"
         )]
         message_format: MessageFormatOpts,
+
+        /// Only list the test binaries, without querying them for the tests they contain.
+        #[clap(long)]
+        no_query: bool,
```

Comment:
I've been thinking about this config option a bit and I think I'd be happier with it as a `--list-type` option, because the JSON output changes quite significantly based on the type.
* `full` (default; run binaries to query and list tests)
* `binaries-only` (do not run binaries)

I think a `--no-query` option doesn't quite explain this fully, but having `type` in the name of the option makes it clearer.